async-nats/src/jetstream/stream.rs

```diff
@@ -139,6 +141,23 @@ impl Stream {
         }
     }
 
+    pub(crate) async fn get_last_message<S: AsRef<str>>(
```

Comment:
Isn't this just get_last_raw_message_by_subject?

Reply:
it is. I guess it was not yet in when I worked on that part :)